arm/Microsoft.ApiManagement/service/products/apis/deploy.bicep

```diff
@@ -0,0 +1,29 @@
+@description('Required. The name of the of the Api Management service.')
+param apiManagementServiceName string
+
+@description('Required. The name of the of the Product.')
+param productName string
+
+@description('Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered')
+param cuaId string = ''
+
+@description('Required. Name of the product api.')
+param name string
+
+module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
+  name: 'pid-${cuaId}'
+  params: {}
+}
+
+resource api 'Microsoft.ApiManagement/service/products/apis@2020-06-01-preview' = {
+  name: '${apiManagementServiceName}/${productName}/${name}'
+}
+
+@description('The resourceId of the product api')
+output apiResourceId string = api.id
+
+@description('The name of the product api')
+output apiName string = api.name
```

Comment:
Now I'm in a pickle. Do I ask you to change this to `output apiName string = last(split(api.name,'/'))` or not? :D

Reply:
Let's push it back for later.